TypeCobol/Compiler/Parser/CodeElementBuilder/CobolExpressionsBuilder.cs

```diff
@@ -311,7 +312,42 @@ internal StorageArea CreateFunctionIdentifier(CodeElementsParser.FunctionIdentif
         [NotNull]
         internal FunctionCall CreateIntrinsicFunctionCall(CodeElementsParser.IntrinsicFunctionCallContext context) {
             var name = CobolWordsBuilder.CreateIntrinsicFunctionName(context.IntrinsicFunctionName());
-            return new IntrinsicFunctionCall(name, CreateArguments(context.argument()));
```

Comment:
The previous method `CreateArguments` should not be called anymore right ?

Reply:
The method `CreateArguments ` is still needed by `CreateUserDefinedFunctionCall`.

Reply:
Ok then there should be some factorization to do between the two.

Reply:
Done